content/en/serverless/azure_app_services/azure_app_services_container.md

```diff
@@ -9,144 +9,162 @@ further_reading:
+COPY . .
 
-{{% svl-init-dotnet %}}
+# Build the application
+RUN dotnet publish -c Release -o out
 
-{{< /programming-lang >}}
-{{< programming-lang lang="ruby" >}}
+# Stage 2: Create a runtime image
+FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS runtime
+WORKDIR /app
 
-{{% svl-init-ruby %}}
+# Copy the build output from stage 1
+COPY --from=build /app/out ./
 
-{{< /programming-lang >}}
-{{< programming-lang lang="php" >}}
+# Datadog specific
+RUN mkdir -p /datadog/tracer
+RUN mkdir -p /home/LogFiles/dotnet
 
-{{% svl-init-php %}}
+ADD https://github.com/DataDog/dd-trace-dotnet/releases/download/v2.49.0/datadog-dotnet-apm-2.49.0.tar.gz /datadog/tracer
+RUN cd /datadog/tracer && tar -zxf datadog-dotnet-apm-2.49.0.tar.gz
 
+# Set the entry point for the application
+ENTRYPOINT ["dotnet", "<your dotnet app>.dll"]
+```
 {{< /programming-lang >}}
 {{< /programming-lang-wrapper >}}
 
-### 2. Configure your application
+Then, build the image and push it to your preferred container registry.
 
-Once the container is built and pushed to your registry, the last step is to set the required environment variables for the Datadog Agent:
-- `DD_API_KEY`: Datadog API key, used to send data to your Datadog account. It should be configured as a [Azure Secret][7] for privacy and safety issue.
-- `DD_SITE`: Datadog endpoint and website. Select your site on the right side of this page. Your site is: {{< region-param key="dd_site" code="true" >}}.
-- `DD_TRACE_ENABLED`: Set to `true` to enable tracing.
+### Create your Linux Web App
 
-For more environment variables and their function, see [Additional Configurations](#additional-configurations).
+1. In the Azure Portal, go to **App Services** and select **Create**.
+1. On the **Basics** tab, provide the required details.
+   - For **Publish**, select **Container**.
+   - For **Operating System**, select **Linux**.
+   Then, select **Next: Container >**.
+1. On the **Container** tab, provide the required details.
+   - For **Sidecar support**, select **Enabled**.
+   - For **Image Source**, select your chosen registry.
+   - Then, specify the **Registry**, **Image**, **Tag**, and **Port** for your container image.
+1. Select **Review + create**, then select **Create**.
 
-### 3. Results
+### Add Datadog environment variables
 
-Once the deployment is completed, your metrics and traces are sent to Datadog. In Datadog, navigate to **Infrastructure->Serverless** to see your serverless metrics and traces.
+In the Azure portal, select your app. In the left menu, select **Configuration** > **Application settings**. Then, add the following environment variables as application settings.
 
-## Deployment
 
-{{% aas-workflow-linux %}}
+`DD_API_KEY` 
+: Your [Datadog API key][3]. <br/>
+**Required**. Alternatively, you can source your API key (and other sensitive information) from Azure Key Vault. See [Use Key Vault references as app settings in Azure App Service][4].
 
-## Additional configurations
+`DD_SITE` 
```

Comment:
This defaults to `datadoghq.com` and only is required if they are on `eu`, `us3` or `us5`